packages/common/src/initiative-templates/_internal/InitiativeTemplateSchema.ts

```diff
@@ -12,5 +12,7 @@ export const InitiativeTemplateSchema: IConfigurationSchema = {
   ...HubItemEntitySchema,
   properties: {
     ...HubItemEntitySchema.properties,
+    previewUrl: { type: "string", format: "url" },
```

Comment:
I only know this because I was working on implementing it concurrently with my scaffolding PR, but I _think_ we're both going to want to do something like this: 
```ts
previewUrl: {
  type: "string",
  if: { minLength: 1 },
  then: { format: "uri" },
}
```
so it doesn't show the format error when the user clears the field

And then in the corresponding `uiSchema` config, you'll need to hide the extra error message from the minLength: 
```ts
{
  scope: "/properties/previewUrl",
  ...
  options: {
    ...
    messages: [
      {
        type: "ERROR",
        keyword: "if",
        hidden: true,
      },
    ],
  },
},
```